wsd/ClientRequestDispatcher.cpp

```diff
@@ -1308,10 +1310,16 @@ bool ClientRequestDispatcher::handleWopiAccessCheckRequest(const Poco::Net::HTTP
             (void) this; // to make the compiler happy wrt. the lambda capture
 #endif
 
-            sendResult(socket, status);
+            std::shared_ptr<StreamSocket> destSocket = socketWeak.lock();
+            if (!destSocket)
+            {
+                LOG_ERR("Invalid socket while sending wopi access check result");
```

Comment:
Good to log the URI, otherwise, it becomes hard to guess which document this is related to (assuming warning/error levels, in which case this might be the first/only entry we see for the given document).